templates/rhsm-katello-reconfigure.erb

```diff
@@ -100,8 +100,8 @@ fi
 
 
 FQDN=`hostname -f`
-if [ $? == "0" ] && [ -d /etc/rhsm/facts/ ]; then
-  echo "{\"network.hostname-override\":\"$FQDN\"}" > /etc/rhsm/facts/katello.facts
+if [ $? == "0" ] && [ "$FQDN" != "localhost" ] && [ -d /etc/rhsm/facts/ ]; then
+  echo "{\"network.hostname-override\":\"$FQDN\"}" > /etc/rhsm/faacts/katello.facts
```

Comment:
I doubt this line was intentional.

![](http://annotateddomini.files.wordpress.com/2012/07/khaaan.jpg)